### PR TITLE
Removes positive stage speed threshold from DNA saboteur 

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Removes the threshold which allowed the DNA saboteur symptom to give positive mutations. 

## Why It's Good For The Game

Being able to sidestep a part of medbay (genetics) with a single disease is bad.

## Changelog
:cl:
tweak: DNA saboteur can no longer give positive mutations
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
As the person who added this threshold i approve this pr. It was buffed a little while ago, which made it render genetics obsolete. Before that, no one used this threshold anyways.

**Review by zeskorion — CHANGES_REQUESTED** [review]
The severityset proc and the desc need to be updated as well

**Comment by leestriter** [comment]
I was the one who wanted it buffed before but I see your point. It came out stronger then I expected, and replacing genetics is bad. I'd prefer we just removed some of the best positive mutations from its list instead of removing it but either way works,. Like zesko said the old version wasn't really used and this one is OP.

**Review by Archanial — DISMISSED** [review]
cool

File: code/datums/diseases/advance/symptoms/genetics.dm [hunk 1]
```diff
@@ -66,8 +63,6 @@ Bonus
 		symptom_delay_max = 60
 	if(A.resistance >= 8) //mutate twice
 		power = 2
-	if(A.stage_rate >= 14)
-		possible_mutations = (GLOB.good_mutations) //This should never have contained not good mutations
 	else
```
Comment (PowerfulBacon) [inline code/datums/diseases/advance/symptoms/genetics.dm]: Bad else

**Review by utilbird — APPROVED** [review]

**Pull request merged**